### ISSUE TITLE
Take into account the search pattern when running all specs in the interactive mode

### ISSUE THREAD

**Issue by bahmutov** [cypress-io/cypress#6581]
Our users are asking us sometimes to run some specs when using the interactive mode. I think the simplest way we could make them happy is to use the search pattern when the user clicks "Run All Specs" button.

<img width="800" alt="Screen Shot 2020-02-27 at 9 18 09 AM" src="https://user-images.githubusercontent.com/2212006/75453683-42b39380-5942-11ea-9e4a-42aa8f2f748e.png">

**Comment by jennifer-shehane** [comment]
Duplicate of #3607 

Albeit a very different suggested implementation - the goal of that request is the same, to run a 'subset' of spec files during `cypress open`.

**Comment by samtsai** [comment]
> Duplicate of #3607
> 
> Albeit a very different suggested implementation - the goal of that request is the same, to run a 'subset' of spec files during `cypress open`.

I've tried the filter and then "Run All Specs" but it seems to just run all the specs ignoring the filter.

**Comment by jennifer-shehane** [comment]
@samtsai Yes, this feature does not exist. https://github.com/cypress-io/cypress/issues/3607 is a request for this feature and has not been released.

**Comment by samtsai** [comment]
> @samtsai Yes, this feature does not exist. #3607 is a request for this feature and has not been released.

Thanks for the clarification @jennifer-shehane , keep up the good work!

**Comment by jennifer-shehane** [comment]
Reopening as I think this doesn't exactly cover the use cases of #3607.
